ttnn/cpp/ttnn/operations/moreh/moreh_bmm/moreh_bmm.hpp

```diff
@@ -0,0 +1,23 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/decorators.hpp"
+#include "ttnn/operations/moreh/moreh_matmul/device/moreh_matmul_device_operation.hpp"
```

Comment:
Please remove this include from .hpp and move to .cpp
Header does not use anything from here.

You might need to include the `compute_kernel_config.hpp` (https://github.com/tenstorrent/tt-metal/blob/main/ttnn/cpp/ttnn/operations/core/compute_kernel/compute_kernel_config.hpp)